### ISSUE TITLE
Is there any documentation or tutorials about this library?

### ISSUE THREAD

**Issue by yangj257** [davidcaron/pclpy#18]
Except for the very simple example, I didn't find any documentation or tutorials about this library. Can you edit one just like laspy, which allows beginners to get started quickly and make your project get more attention. You can start with simple modules, like I/O,kd-tree,octree,visualization,etc. Thank you very much!

**Comment by davidcaron** [comment]
Yes, it's definitely needed. My focus was on getting it to work first. I'll write a first draft when I get a chance.

**Comment by yangj257** [comment]
I hope that the first draft can be completed as soon as possible, I can't wait for it!
You have contributed a lot to python and pcl!

**Comment by pournami123** [comment]
@davidcaron Is there any documentation available?  please share

**Comment by ellie-b** [comment]
I would also like to see documentation to get started using this @davidcaron

**Comment by OloOcki** [comment]
up! :)

**Comment by ellie-b** [comment]
@OloOcki Where is the additional documentation?

**Comment by OloOcki** [comment]
There is no additional documentation. That is why I wrote "up!" ;)

**Comment by soulslicer** [comment]
Yet another library that lacks proper documentation. Just like strawberry labs garbage pcl port, pptk, and open3d. all lack even being able to do a simple thing like visualzing a color pointcloud in real time in python.

I ended up writing by own pybind11 bindings to the PCL C++ API and am using that now.

**Comment by glassTransition** [comment]
> I ended up writing by own pybind11 bindings to the PCL C++ API and am using that now.

I really don't understand why these develops bother to "provide" bindings if they have no intention of making the libraries usable and not providing any other form of support. Open3d's forum is a windtunnel.

I wish I had the skills to write my own bindings, but I'm trying to use this library because I only know Python. I don't have the time to learn C++
